db/schema.rb

```diff
@@ -12,21 +12,21 @@
 
 ActiveRecord::Schema.define(version: 2020_12_10_084736) do
 
-  create_table "analytics", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "analytics", charset: "utf8", force: :cascade do |t|
```

Comment:
Does this require any kind of migration at all?

Reply:
it does not